portal-web/test/functional/com/liferay/portalweb/macros/CompanyLogging.macro

```diff
@@ -68,7 +68,7 @@ definition {
 		return ${logContent};
 	}
 
-	@summary = "Default summary"
+	@summary = "This will get the Company log ${filePath}"
```

Comment:
We are defining the company instance name and returning the filePath. This should be
"This will get the company log path for company ${portalInstanceName}